airbyte_cdk/manifest_migrations/migrations/registry.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+#
+
+manifest_migrations:
+  - version: 6.45.2
+    migrations: 
+      - name: http_requester_url_base_to_url
+        order: 1
+        description: |
+          This migration updates the `url_base` field in the `http_requester` spec to `url`.
+          The `url_base` field is deprecated and will be removed in a future version.
+      - name: http_requester_path_to_url
+        order: 2
```

Comment:
❓ Ive been out of the loop, so sorry for the dumb question.

Previously the version, and its order was infered by the migration class name. e.g. `v1_1_1_0_NAME_OF_MIGRATION.py`

Now we are explicitly setting the order in this file.

Just curious what made you change course?